rust_src/src/buffers.rs

```diff
@@ -635,7 +632,7 @@ pub fn barf_if_buffer_read_only(position: Option<EmacsInt>) -> () {
 /// No such buffer error.
 #[no_mangle]
 pub extern "C" fn nsberror(spec: LispObject) -> ! {
-    let spec = LispObject::from_raw(spec);
+    let spec = spec;
```

Comment:
no-op line.

Reply:
Good catch!

Reply:
Yep, this PR definitely opens up some opportunities for refactoring. You are welcome to submit a PR, or I will get around to it at some point :)